Source/Core/InputCommon/GCAdapter.cpp

```diff
@@ -370,8 +367,7 @@ static void Reset()
     s_adapter_output_thread.join();
   }
 
-  for (int i = 0; i < SerialInterface::MAX_SI_CHANNELS; i++)
-    s_controller_type[i] = ControllerTypes::CONTROLLER_NONE;
+  s_controller_type.fill(ControllerTypes::CONTROLLER_NONE)
```

Comment:
You should still add the missing `;` here so it compiles.

Reply:
Ah, I've messed with rebasing.